websites/C/Coub/dist/metadata.json

```
@@ -19,7 +19,7 @@
 		"vi_VN": "Coub là trang web chia sẻ video trực tuyến. Nó cho phép người dùng tạo và chia sẻ video lặp lại lên tới mười giây, sử dụng các video có sẵn tại YouTube, Vimeo, hoặc tệp riêng của họ"
 	},
 	"url": "coub.com",
-	"version": "2.0.15",
+	"version": "2.0.16",
```

You have added functionality, this should be a minor bump instead
```suggestion
	"version": "2.1.0",
```